examples/plot_linear_regression.py

```diff
@@ -0,0 +1,217 @@
+# -- import necessary packages and modules
+from inspect import isgenerator
+from collections import namedtuple
+
+import numpy as np
+from scipy import linalg
+from sklearn.linear_model import LinearRegression
+
+# import pandas as pd
+# import matplotlib.pyplot as plt
+
+# import mne
+from mne.datasets import limo
+
+# from mne.viz import plot_compare_evokeds
+from mne.stats import linear_regression
```

Comment:
Or ridge.

Reply:
does `mne.stats` has ridge? or am I missing something?

Reply:
Meant to go under line 13